documents/invoice/service.go

```diff
@@ -150,8 +151,14 @@ func (s service) calculateDataRoot(old, new documents.Model, validator documents
 		return nil, errors.NewTypedError(documents.ErrDocumentInvalid, err)
 	}
 
+	// get tenant ID
+	tenantID, err := s.config.GetIdentityID()
```

Comment:
lets explicitly pass tenentID through function args. Makes it easier later for multi tenency